ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/member/services/TreeMemberRemoverImpl.java

```diff
@@ -16,7 +16,7 @@ public TreeMemberRemoverImpl(MemberRepository memberRepository) {
 		this.memberRepository = memberRepository;
 	}
 
-	public void tryRemovingMember(String memberId) {
+	public void tryDeletingMember(String memberId) {
 		Optional<Member> member = memberRepository.getMember(memberId);
 		if (member.isPresent() && member.get().getTreeNodeReferences().isEmpty()) {
```

Comment:
I feel like this code missed an opportunity to leverage the advantages of having an optional return value. This could be written as:

`		memberRepository
				.getMember(memberId)
				.filter(member -> member.getTreeNodeReferences().isEmpty())
				.ifPresent(member -> {
					memberRepository.deleteMember(memberId);
					Metrics.counter("ldes_server_deleted_members_count").increment();
				});`